docs/deployment/getting_started_locally_with_extensions.md

```diff
@@ -0,0 +1,160 @@
+
+> There are `.gitignore` entries for all files and directories which include credentials. Nevertheless, please double check and make sure that credentials are not commited.
+
+### DNS
+Gardener control plane requires DNS for default and internal domains. Thus, you have to configure a valid DNS provider for your setup.
+
+Please maintain your DNS provider configuration and credentials at `./example/provider-extensions/garden/controlplane/values.yaml`.
+
+You can find a template for the file at `./example/provider-extensions/garden/controlplane/values.yaml.tmpl`.
+
+### Infrastructure
+Infrastructure secrets and the corresponding secret bindings should be maintained at:
+-  `./example/provider-extensions/garden/project/credentials/infrastructure-secrets.yaml`
+-  `./example/provider-extensions/garden/project/credentials/secret-bindings.yaml`
+
+There are templates with `.tmpl` suffixes for the files in the same folder.
+
+### Seed Cluster Preparation
+The `kubeconfig` of your Kubernetes cluster you would like to use as seed should be placed at `./example/provider-extensions/seed/kubeconfig`.
+Additionally, please maintain the configuration of your seed in `./example/provider-extensions/seed/seed-config.yaml`. You can use `seed-config.yaml.tmpl` in the same directory as a template. This file also includes explanations of the settings.
+
+Using a Gardener cluster as seed simplifies the process, because some configuration options can be taken from `shoot-info` and creating DNS entries and TLS certificates is automated.
+
+However, you can use different Kubernetes clusters for your seed too and configure these things manually. Please configure the options of `seed-config.yaml` upfront. For configuring DNS and TLS certificates, `make gardener-extensions-up` , which is explained later, will pause and tell you what to do.
+
+### `CloudProfile`s
+There are no demo `CloudProfiles` yet. Thus, please copy `CloudProfiles` from another landscape to `./example/provider-extensions/garden/cloud-profiles/profiles` or create your own `CloudProfiles` based on the [gardener examples](../../example/30-cloudprofile.yaml). Please check the GitHub repository of your desired provider-extension. Most of them include example `CloudProfile`s.
+
+## Setting Up The KinD Cluster
+
+```bash
+make kind-extensions-up
+```
+
+This command sets up a new KinD cluster named `gardener-local` and stores the kubeconfig in the `./example/provider-extensions/garden/kubeconfig` file.
+
+> It might be helpful to copy this file to `$HOME/.kube/config` since you will need to target this KinD cluster multiple times.
+Alternatively, make sure to set your `KUBECONFIG` environment variable to `./example/gardener-local/kind-extensions/kubeconfig` for all future steps via `export KUBECONFIG=example/gardener-local/kind-extensions/kubeconfig`.
```

Comment:
Are we sure that `example/gardener-local/kind-extensions/kubeconfig` is the correct path? On my machine the path was `example/gardener-local/kind/extensions/kubeconfig`.

Reply:
`example/gardener-local/kind/extensions/kubeconfig` is right. The other one was the status before Rafael refactored the kind setup.